test/ColorJudgeTest .cpp

```diff
@@ -0,0 +1,214 @@
+/**
+ * @file   ColorJudgeTest.cpp
+ * @brief  ColorJudgeクラスをテストする
+ * @author keiya121
+ */
+
+#include "ColorJudge.h"
+#include "RgbData.h"
+#include <gtest/gtest.h>
+
+namespace etrobocon2024_test {
+  TEST(ColorJudgeTest, getColorZeroRgb)
```

Comment:
```suggestion
  // RGB値が全て0の色を取得し、色を判定する
  TEST(ColorJudgeTest, getColorZeroRgb)
```
他のファイルはあるので、そっちに合わせます。